### PR TITLE
fix: XAI rebrand

### PR DESCRIPTION
**Pull request by andrewferlitsch**

### CODE REVIEW

**Comment by ivanmkc** [comment]
Quickly looked over and it looks good.

You just need a TW approval.

**Comment by ivanmkc** [comment]
Seems like you still need a TW approval.

**Review by thedriftofwords — APPROVED** [review]
LGTM on the rebranding. Let's handle the notebook rewrites in subsequent PRs.

File: ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb [hunk 1]
```diff
@@ -0,0 +1,2673 @@
+{
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb:1]: <p>Can this be replaced with:</p><p> </p><p>REGION = '[your-region]'&nbsp;#@param {type: "string"}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Comment by andrewferlitsch** [comment]
fixed region change

**Pull request merged**